datadotworld/__init__.py

```diff
@@ -15,5 +15,8 @@
 permissions and limitations under the License.
 
 This product includes software developed at data.world, Inc.(http://www.data.world/).
-'''
+"""
```

Comment:
my only reason for the `'''` instead of `"""` was the single `"`s on line 12 - I used these doc strings in R, and it is NOT tolerant of that, but apparently Python is, so 👍